test/remote/gateways/remote_braintree_blue_test.rb

```diff
@@ -199,6 +199,14 @@ def test_successful_credit_card_verification
     assert_equal 'P', response.avs_result['code']
   end
 
+  def test_failed_credit_card_verification
+    credit_card = credit_card('378282246310005', verification_value: '544')
+
+    assert response = @gateway.verify(credit_card, @options.merge({ allow_card_verification: true }))
+    assert_failure response
+    assert_match "CVV must be 4 digits for American Express and 3 digits for other card types. (81707)", response.message
```

Comment:
@Carrigan I can actually remove this line with the message match, the assert_failure alone has the same change in behavior on master vs this branch. What do you think?

Reply:
I think the remote test is fine, it might just be good to have an additional test with a crystalized response so that it is more repeatable and obvious what type of error this catches

Reply:
Totally 👍  updated!